### ISSUE TITLE
Update Module for Foundry VTT v11 Compatibility

### ISSUE THREAD

**Issue by brunocalado** [Javiondox/html-to-scene#23]
Dear maintainer,

Could you please update the module to be compatible with Foundry VTT v11? It would greatly enhance our gaming experience and allow us to continue using this wonderful tool. Thank you in advance for your support.

**Comment by avorial** [comment]
Yes, This is a amazing product that lets us do so much that can't be done any other way.

**Comment by Javiondox** [comment]
I’ll make sure to get some free time to update it. I’ll have to relearn Foundry, and I’ll probably rewrite the entire module from scratch.  So I can’t give you a time frame for when it will be done.

**Comment by Snakeplinsky** [comment]
Yes , is my favorite module to foundry.

**Comment by pinkflumph** [comment]
For anyone else who is looking for similar functionality, the [Inline Webviewer](https://foundryvtt.com/packages/inlinewebviewer) module can also create HTML scenes and it works with v11!

**Comment by alexadrerui** [comment]
any news for update to v12?

**Comment by Snakeplinsky** [comment]
I just updated Foundry VTT to v12 and now html-to-scene doesn't work anymore.
Help

**Comment by Plasma-AndraaX** [comment]
Here a fix : https://github.com/Javiondox/html-to-scene/pull/24
